src/App.tsx

```diff
@@ -49,8 +50,8 @@ function App() {
     <div className="App" >
       <Grid container direction="column">
         <Grid item container>
-          <Grid item xs={false} sm={false} md={2} lg={2} xl={2} />
-          <Grid item xs={12} sm={12} md={8} lg={8} xl={8} >
+          <Grid item xs={false} sm={false} md={false} lg={1} xl={1} />
```

Comment:
empty grid?

Reply:
used to center review container in laptop & xl screens